README.adoc

```diff
@@ -0,0 +1,253 @@
+
+The code of the `{repo}` repo is under v3 of the link:LICENSE.adoc[GPL]. It is similar to the Golden Rule: do unto others as you would have them do unto you. In exchange for benefitting from the work completed in this repo, others must share their derivative work under v3 of the link:LICENSE.adoc[GPL].
+
+> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+> This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+> You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+## Gurbani and Panthic Compilations
+
+IMPORTANT: Applies to different texts inside the `data` folder and generated inside the `build` folder.
+
+As typical of many old, historical works, most gurbani and panthic texts are free of known copyright restrictions. We identify it as being in the https://creativecommons.org/publicdomain/mark/1.0/[public domain] as a work of factual compilation with originality. The honor and reputation of the original works are to be maintained. Derogatory treatments (including adding to, deleting from, altering of, or adapting) the words in a way that distorts or mutilates the original work is forbidden. That is why, whenever possible, physical sources are used to determine the digital representation of these works as to avoid any controversy.
+
+Citations for accuracy and review are based on MLA formatting. More often than not, a condensed version of the following is used when citing changes to any of the sources in the database:
+
+```
+Author. Title of source. Title of container, Other contributors, Version, Number, Publisher, City of Publication, Publication Date, Location. Date of access.
+```
+
+### Sri Guru Granth Sahib Ji
+
+```
+Shabadaarth. Vol. 1-4, SGPC, Sri Amritsar, 2009-2012, www.discoversikhism.com/sikh_library/gurmukhi/shabdaarth_sri_guru_granth_sahib_ji_part_1.html. Accessed 22 Jan 2019.
+```
+
+Vishraam / Pauses:
+```
+Dauder, Jaswant S. Sri Guru Granth Sahib Ji Shudh Ucharan, Arth Paath Vishraam Pothi, Sri Damdami Taksaal. Oct 2015, www.gursevak.com/drupal7/sites/default/files/Vishram%20Sri%20Guru%20Granth%20Sahib%20Jee.pdf. Accessed 18 May 2020.
+```
+
+Arth / Translations:
+```
+Randhawe, Hari S. Gurbani Arth-Bhandar. 1st ed., Vol. 1-12, Khalsa Charitable Trust, Fatehgarh Sahib, 2016-2017, www.discoversikhism.com/sikh_library/gurmukhi/gurbani_arth_bhandar-volume_01.html. Accessed 18 May 2020.
+
+Singh, Sahib. Sri Guru Granth Darpan. Vol. 1-10, Raaj Publishers, Hoshiarpur/Jalandhar, 1972, www.sikhbookclub.com/Book/Sri-Guru-Granth-Sahib-Darpan-Pothi-Pehli. Accessed 18 May 2020.
```

Comment:
We are currently using the softcopy: http://www.gurbanifiles.net/gurmukhi/GuruGranth%20Darpan%20by%20Prof%20Sahib%20Singh.pdf

https://archive.org/details/SriGuruGranthSahibDarpan-Volume1
https://archive.org/details/SriGuruGranthSahibDarpan-Volume3
https://archive.org/details/SriGuruGranthSahibDarpan-Volume4
https://archive.org/details/SriGuruGranthSahibDarpan-Volume5
https://archive.org/details/SriGuruGranthSahibDarpan-Volume6
https://archive.org/details/SriGuruGranthSahibDarpan-Volume7
https://archive.org/details/SriGuruGranthSahibDarpan-Volume8
https://archive.org/details/SriGuruGranthSahibDarpan-Volume9
https://archive.org/details/SriGuruGranthSahibDarpan-Volume10

Reply:
Updaetd link